courses/free-courses-hi.md

```diff
@@ -85,6 +97,12 @@
 * [DevOps Tutorials in Hindi/Urdu \| Devops सीखने का सबसे आसान तरीका \| Complete Devops including Git,Jenkins,Maven,Chef,docker,Ansible and Kubernetes](https://www.youtube.com/playlist?list=PLBGx66SQNZ8aPsFDwb79JrS2KQBTIZo10) - Bhupinder Rajput, Technical Guftgu
 
 
+### Flutter
+
+* [Complete Flutter Tutorial In Hindi By Desi Programmer](https://youtube.com/playlist?list=PLlFwzkUNmr94BF0KH7BYPL7DsZjhJRdTm) - Desi Programmer
+* [Flutter Tutorial For Beginners in Hindi](https://youtube.com/playlist?list=PLMkkZSS5OjPIwDyHHKVex6zr008U1-sWM) - Geeks Rank
```

Comment:
Patch to recover a diff found with #7005 at commit 9fadbf98af17205729f3e562a4fcbd610b83bdf0 but alphabetizing Flutter list: https://github.com/EbookFoundation/free-programming-books/pull/7005/commits/9fadbf98af17205729f3e562a4fcbd610b83bdf0#diff-0bed9be5a9c29db386438a12b2e1abaac6945680b2e2ab9d07c0b58063e315adR93
```suggestion
* [Flutter 3 tutorial in Hindi](https://youtube.com/playlist?list=PLB97yPrFwo5g-XcPlfSXSOeeby23jVAcp) - CODERS NEVER QUIT
* [Flutter Tutorial For Beginners in Hindi](https://youtube.com/playlist?list=PLMkkZSS5OjPIwDyHHKVex6zr008U1-sWM) - Geeks Rank
```